src/components/TrackTile/PillarsTile/Pillar.tsx

```diff
@@ -381,6 +381,7 @@ const defaultStyles = StyleSheet.create({
    * @property {string} color - Set to 'currentValue' to use the tracker's color
    */
   pillarIconGoalNotMet: {
-    color: 'white',
+    // theme.colors.onSurfaceDisabled
```

Comment:
Thanks for being in speed mode (in response to me asking for it), but let's not leave commented-out line in as-is.  If we want to keep them in place for some reason, please add a TODO: prefix to each section.  Kind of annoying, but I've just seen commented-out code last years too many times.